<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSpiNNaker</class>
 <widget class="QWidget" name="NewSpiNNaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_v_reset">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>300</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tau_syn_I</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tau_m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cm">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tau_syn_E</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>v_rest</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tau_syn_I">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>270</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tau_syn_E">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_v_rest">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>330</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_i_offset">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tau_refrac">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_refrac">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tau_refrac</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neuron Data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tau_m">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>i_offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>v_reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export to SpiNNaker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>v_thresh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_v_thresh">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fileName">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>490</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_timeStep">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>390</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>timeStep</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_delay">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>420</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>454</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Sim.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_timeSimulation">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>450</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Neurons">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neurons</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Neuron">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Neurons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Label</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ip</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_plot">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate plot code</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_times">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>111</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
